src/transformStateWithClones.js

```diff
@@ -6,8 +6,37 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = JSON.parse(JSON.stringify(state));
```

Comment:
could have been simpler
```suggestion
  const stateCopy = { ...state };
```